app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,7 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.11.1/mapbox-gl.css" rel="stylesheet" />
```

Comment:
Is this necessary with the `globalMapStyle.jsx` you set up with @ericgibb?

Reply:
According to the API, yes : https://docs.mapbox.com/mapbox-gl-js/api/
_Include the JavaScript and CSS files in the <head> of your HTML file. The CSS file is required to display the map and make elements like Popups and Markers work._
- <link href='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.css' rel='stylesheet' />

Without this line included, the map doesn't render properly - and the 'markers' (bus pins) won't appear at all.